io.naox.InnerBreeze.json

```diff
@@ -0,0 +1,37 @@
+{
+  "app-id": "io.naox.InnerBreeze",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "copy-icon": "true",
+  "finish-args": [
+      "--share=ipc",
+      "--share=network",
+      "--socket=wayland",
+      "--socket=fallback-x11",
+      "--device=dri"
+  ],
+  "command": "inner_breeze",
+  "modules": [
+      {
+          "name": "InnerBreeze",
+          "buildsystem": "simple",
+          "build-commands": [
+              "chmod +x inner_breeze",
+              "install -D inner_breeze /app/bin/inner_breeze",
+              "cp -r data /app/data",
+              "cp -r lib /app/lib",
+              "install -Dpm644 extra/io.naox.InnerBreeze.desktop /app/share/applications/io.naox.InnerBreeze.desktop",
+              "install -Dpm644 extra/io.naox.InnerBreeze.metainfo.xml /app/share/metainfo/io.naox.InnerBreeze.metainfo.xml",
+              "install -Dpm644 extra/InnerBreeze.png /app/share/icons/hicolor/256x256/apps/InnerBreeze.png"
```

Comment:
this is the wrong icon name. I'm actually surprise the build doesn't complain.

Reply:
hi thanks for the feedback.

why is it wrong? that's what it's called now in the archive, does it have to be named in a specific way?